samples/SocialSample/Startup.cs

```diff
@@ -136,7 +136,7 @@ k web
                         JObject user = JObject.Parse(text);
 
                         var identity = new ClaimsIdentity(
-                            context.Options.AuthenticationType,
+                            context.Options.AuthenticationScheme,
```

Comment:
Should we add a new option like ClaimIssuer to AuthenticationOptions and set defaults appropriately instead of just hardcoding?